src/i_oplmusic.c

```diff
@@ -1774,11 +1774,12 @@ static boolean OPL_InitMusic(void)
     return true;
 }
 
-int I_OPL_DeviceList(const char* devices[], int size)
+static int I_OPL_DeviceList(const char* devices[], int size, int *current_device)
 {
+    *current_device = 0;
     if (size > 0)
     {
-        devices[0] = "OPL";
+        devices[0] = "OPL Emulation";
```

Comment:
While we're at explicit long device names, strictly speaking it's "OPL3 Emulation" here...

Reply:
We can add an OPL2 device, although it doesn't make much of a difference to me.

Reply:
In fact, we could offer both OPL2 and OPL3 emulation with the code we have in the repo. 🤔

Reply:
Sure. I meant that we can also offer several devices for OPL emulation. Maybe that's too much, I already have ~20 items in my list of MIDI players 😄 Though most users will probably have ~5.

Reply:
Wow, lol! No, I think you are right and nobody will ever miss OPL2 emulation.